jwql/website/apps/jwql/templates/engineering_database.html

```diff
@@ -0,0 +1,130 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+	<title>DMS Engineering Database - JWQL</title>
+
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="container">
+    	<h2>Explore mnemonics in the DMS Engineering Database</h2>
+        <hr>
+
+
+        <br>
+        This page allows interactive exploration of mnemonics stored in the Engineering Database (EDB) powered by MAST. <br>
+
+        Its use requires access privileges to the EDB (see <a href="https://outerspace.stsci.edu/pages/viewpage.action?spaceKey=JARI&title=JWST+Archive+Rehearsal+Instructions">here</a>)
```

Comment:
@Johannes-Sahlmann It think it would be better if the links here opened a new tab in the browser instead of redirect the user away from the `jwql` site altogether.  I think this can be accomplished by supplying a `target` parameter, or something like that.

Reply:
Yes, just add `target="_blank"`!